js/main.js

```diff
@@ -0,0 +1,29 @@
+//Функция, возвращающая случайное целое число из переданного диапазона включительно.
+
+function randomInteger(min, max) {
+    // случайное число от min до (max+1)
+    let rand = min + Math.random() * (max + 1 - min);
+    return Math.floor(rand);
+}
+
+alert(randomInteger(1, 10));
```

Comment:
Можно просто вызвать функцию, `alert`'ы все таки не самое удобное средство дебага, т.к. блокирует пользовательское взаимодействие.
```suggestion
console.log(randomInteger(1, 10));
```

Reply:
+ поправила